dashboard/test/ui/features/spritelab/dancelab.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Dance Lab
+  Scenario: Without flag, song selector is not visible
+    Given I am on "http://studio.code.org/s/allthethings/stage/37/puzzle/1?noautoplay=true"
+    And I rotate to landscape
+    And I wait for the page to fully load
+    And I wait for 3 seconds
```

Comment:
@joshlory what are we waiting to have happen here? would be nice to wait explicitly for this before this pattern gets copied too many times. Erin that shouldn't block your PR though